docs/user_guides/debugging/debugging_environment.rst

```diff
@@ -28,7 +27,7 @@ You will also need to create and activate conda environment with installed `numb
 .. note::
 
     Known issues:
-      - Debugging tested with following packages: ``numba-dppy=0.14``, ``dpctl=0.8``, ``numba=0.53``.
+      - Debugging features were tested with following packages: ``numba-dppy=0.14``, ``dpctl=0.8``, ``numba=0.53``.
```

Comment:
Why is this an issue? It is already inside a Note and I think that is enough. All we are trying to tell the user that we tested only for a specific version of the software and a their mileage may vary.

Reply:
Fixed